src/Feldspar/Onnx/onnxToFeld.hs

```diff
@@ -29,67 +29,111 @@
 
 module Main where
 
-import Onnx.ModelProto
-import Onnx.GraphProto as G
-import Onnx.NodeProto as N
-import Onnx.ValueInfoProto as V
-import Onnx.AttributeProto as A
-import Onnx.TypeProto as T
-import Onnx.TypeProto.Value as TV
-import Onnx.TypeProto.Tensor as TT
-import Onnx.TypeProto.Sequence as TS
-import Onnx.TypeProto.Map as TM
-import Onnx.TensorProto as TP
-import Onnx.TensorProto.DataType as TD
-import Onnx.TensorShapeProto as TSP
-import Onnx.TensorShapeProto.Dimension as TSP
-import Onnx.TensorShapeProto.Dimension.Value as TSP
-
-import Text.ProtocolBuffers
+import qualified Onnx.ModelProto as O
+import qualified Onnx.GraphProto as G
+import qualified Onnx.NodeProto as N
+import qualified Onnx.ValueInfoProto as V
+import qualified Onnx.AttributeProto as A
+import qualified Onnx.TypeProto as T
+import qualified Onnx.TypeProto.Value as TV
+import qualified Onnx.TypeProto.Tensor as TT
+import qualified Onnx.TensorProto as TP
+import qualified Onnx.TensorProto.DataType as TD
+import qualified Onnx.TensorShapeProto as TSP
+import qualified Onnx.TensorShapeProto.Dimension as TSP
+import qualified Onnx.TensorShapeProto.Dimension.Value as TSP
+
+import Text.ProtocolBuffers (messageGet)
 import Text.ProtocolBuffers.Header as H (Utf8(..))
 
 import qualified Data.ByteString.Lazy as L
 import qualified Data.ByteString.Lazy.UTF8 as U
+import qualified Data.ByteString.Builder as B
 
-import Data.Either (either)
-import qualified Data.Foldable as D (toList)
+import qualified Data.Foldable as D (toList, foldMap)
+import qualified Data.Set as S
```

Comment:
This should be after Data.Maybe, right?

Reply:
I think mixing qualified and unqualified imports is an eye-sore, so I prefer sorting each kind by itself.